README.md

```diff
@@ -876,12 +876,28 @@ Java 7 or above is required for using the clients in this repository.
 Supported Platforms
 -------------------
 
-This client is supported on Mac OS X, Windows and Linux (excluding Android and Alpine).
-Google Cloud Platform environments currently supported include GCE, GKE and GAE Flex.
-GAE Standard is not currently supported.
+Clients in this repository use either HTTP or gRPC for the transport layer. All
+HTTP-based clients should work in all environments (theoretically).
```

Comment:
I like the "theoretically" part in the readme, but probably it should not be there =). Otherwise it looks like we ourselves don't know how our libraries work (which is true, but it should remain a secret...)